### ISSUE TITLE
Push money to rubygems or package cloud as shopify_money

### ISSUE THREAD

**Issue by Senjai** [Shopify/money#129]
It'd be nice to rename the gem to ShopifyMoney but we can't really do this because of the dependencies that people may have on the project. That said we always have to git source because we don't have this on a gem server anyway as it will conflict with the ruby "money" gem.

If our intent is to keep maintaining a fork, we should probably add support for pushing this as a gem called "shopify_money" and deprecate the scenario who have this listed as a dependency with money.

We run into a possible scenario where people may include both, but it's better to have a proper versioning scheme to get the benefit of regular dependency management.

@jasonwebster @bdewater

**Comment by agarcher** [comment]
Any update on this? Lack of availability on package cloud is preventing several channels from removing git sourced gems (see https://github.com/Shopify/messenger-commerce/issues/2359, https://github.com/Shopify/pinterest-commerce/issues/3381 and https://github.com/Shopify/facebook-commerce/issues/7495).

**Comment by Senjai** [comment]
@agarcher No update just yet. I created the issue to make sure our conversation on it wasn't lost and that so it could be tackled when someone had the bandwidth. I currently don't have this at the moment though.

Here's a breakdown of the conversations I did have though:

With respect to how we'd actually affect the change, we talked about having two gemspecs such that the gem would still work for git sourced Gemfiles, and then publish them gem on package cloud under a different name so that it would no longer compete or be confused with the ruby version of money. We would then swap over all relevant projects and remove the ability to gitsource the gem as it currently stands.

It was hinted that we could use ServicesDB in some fashion to identify repositories using the money gem and once we fix the package cloud issue we could automate the creation of pull requests across those repositories, but I've not looked into it past this.

I don't actually know if this is possible or not, but there's alternative ways of doing the same thing if we had to for a transition.

cc @Krystosterone & @jasonwebster
